### PR TITLE
note style

### PR DESCRIPTION
**Pull request by carolinp**

### CODE REVIEW

**Comment by carolinp** [comment]
screen shot 2018-05-10 at 10 29 49 am
screen shot 2018-05-10 at 10 29 33 am

File: layouts/_default/single.html [hunk 1]
```diff
@@ -143,7 +147,6 @@ <h6 class="dropdown-header list-label general">
       </div>
      </div>
     </div>
-   </section>
```
Comment (carlmjohnson) [inline layouts/_default/single.html]: Does moving the section end affect anything?

File: static/scss/candidate.scss [hunk 2]
```diff
@@ -33,10 +33,35 @@
...
     color: #2e98cc;
   }
+  @media only screen and (max-width: 990px) {
+    margin-top: 0;
+  }
+  @media only screen and (max-width: 770px) {
+    padding: 1px 10px 10px 10px;
+  }
+}
+.section-header.note-header {
```
Comment (carlmjohnson) [inline static/scss/candidate.scss]: Can this be `.note .note-header` and still work?

File: static/scss/candidate.scss [hunk 3]
```diff
@@ -33,10 +33,35 @@
...
+  text-align: center;
+  color: #a5a5a5;
+  padding-left: 0;
+}
+
+.note-body {
```
Comment (carlmjohnson) [inline static/scss/candidate.scss]: Make this `.note .note-body`. We're not using `.note-body` anywhere else, but putting it under `.note` makes it so we don't have to worry about conflicts.

File: static/scss/candidate.scss [hunk 4]
```diff
@@ -33,10 +33,35 @@
...
     font-style: oblique;
   }
   a {
     color: #2e98cc;
   }
+  @media only screen and (max-width: 990px) {
+    margin-top: 0;
+  }
+  @media only screen and (max-width: 770px) {
+    padding: 1px 10px 10px 10px;
```
[the remark this answers is not in the document]
Comment (carolinp) [reply static/scss/candidate.scss]: It's just a weird workaround for the h2 margin-top -- I can re-do the header styles to have it make more sense.

**Pull request merged**